cron_jobs/region_regen_cron.php

```diff
@@ -0,0 +1,160 @@
+<?php
+session_start();
+
+/**
+ * Region Cron Job
+ **
+ *  This should be processed every hour.
```

Comment:
```suggestion
 *  This should be processed every WarManager::REGEN_INTERVAL_MINUTES
```

Reply:
An idempotent version of this where we track the last time it ran and only apply the correct amount of regen would be awesome (set cron to run every min, define regen interval separately and it will wait until regen interval is passed to trigger regen), but not something we need to do right now